var/ramble/repos/builtin/applications/gromacs/application.py

```diff
@@ -59,6 +59,23 @@ class Gromacs(SpackApplication):
                sha256='f11745201dbb9e6a29a39cb016ee8123f6b0f519b250c94660f0a9623e497b22',
                description='12M Atoms, Peptides in Water, 2fs time step, all bonds constrained. https://www.mpinat.mpg.de/grubmueller/bench')
 
+    input_file('BenchPEP_h', url='https://www.mpinat.mpg.de/benchPEP-h.zip',
```

Comment:
Not strictly due to this PR, but given you're modifying the Gromacs application, can you remove the commented out definition of BenchPEP, since it is repeated?